podcasts/New Search/SearchHistoryView.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+import PocketCastsDataModel
+import PocketCastsUtils
+
+struct SearchHistoryView: View {
+    @EnvironmentObject var theme: Theme
+
+    private var episode: Episode {
+        let episode = Episode()
+        episode.title = "Episode title"
+        episode.duration = 3600
+        return episode
+    }
+
+    init() {
+        UITableViewHeaderFooterView.appearance().backgroundView = UIView()
+    }
+
+    var body: some View {
+        VStack {
+            List {
+                HStack {
+                    Text("Recent searches")
+                        .font(style: .title2, weight: .bold)
+                    Spacer()
+                    Button("Clear all".uppercased()) {}
```

Comment:
Same as to localized question

Reply:
Done in 9566d7d425c0a99cf89f7732f1d8aa817fd34a8b